scio-core/src/main/scala/com/spotify/scio/values/PairSCollectionFunctions.scala

```
@@ -104,7 +105,7 @@ class PairSCollectionFunctions[K, V](val self: SCollection[(K, V)]) {
           .apply("TupleToKv", toKvTransform)
           .setCoder(CoderMaterializer.kvCoder[K, V](context))
           .apply(t)
-        if (!kv.getCoder.isInstanceOf[KvCoder[_, _]]) {
+        if (!(Try(kv.getCoder.isInstanceOf[KvCoder[_, _]]).getOrElse(false))) {
```

Just curious, what exception are you trying to swallow?